src/components/order/Order.tsx

```diff
@@ -122,6 +169,7 @@ const Order: React.FC = () => {
               type="text"
               ref={SenderNameRef}
               placeholder="이름을 입력하세요."
+              defaultValue={user?.name}
             ></input>
```

Comment:
self-closing이 가능할 때는 self-closing을 해주시면 좋을거 같네요
```suggestion
              defaultValue={user?.name}
            />
```